engine/Assets/Scripts/Behaviors/ArcadeDriveBehaviour.cs

```diff
@@ -50,17 +55,22 @@ public class ArcadeDriveBehaviour : SimBehaviour {
             _leftWheels  = leftWheels;
             _rightWheels = rightWheels;
 
+            forward  = SimObjectId + FORWARD;
```

Comment:
This needs to be `MiraLive.MiraAssembly.Info.GUID` so it saves for multiple iterations of the same bot